### PR TITLE
Update for feature_cmsis5

### PR DESCRIPTION
**Pull request by Patater**
- Switch from RTX to RTX2
- Add EFM32GG support
- Clean up a bit

@AlessandroA @niklas-arm

### CODE REVIEW

**Comment by 0xc0170** [comment]
bump

File: mbed_app.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
         "*": {
             "target.features_add": ["UVISOR"],
             "target.extra_labels_add": ["UVISOR_SUPPORTED"]
+        },
+        "EFM32GG_STK3700": {
```
Comment (AlessandroA) [inline mbed_app.json]: This is not needed anymore, because of the `*` in line 3.

File: mbed_app.json [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+            "target.features_add": ["UVISOR"],
+            "target.extra_labels_add": ["UVISOR_SUPPORTED"]
```
Comment (AlessandroA) [inline mbed_app.json]: Please modify `test/filters.json` as well (remove `EFM32GG_STK3700` from the blacklist).

**Review by AlessandroA — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: mbed_app.json (hunk 1 above)
Comment (Patater) [reply mbed_app.json]: Fixed

File: mbed_app.json (hunk 2 above)
Comment (Patater) [reply mbed_app.json]: Updated

**Review by AlessandroA — APPROVED** [review]
LGTM

**Comment by AlessandroA** [comment]
retest uvisor

**Pull request merged**